app/components/assets/form.tsx

```diff
@@ -145,7 +145,7 @@ export const AssetForm = ({
         encType="multipart/form-data"
       >
         <AbsolutePositionedHeaderActions className="hidden md:flex">
-          <Actions disabled={disabled} />
+          <Actions assetId={id} disabled={disabled} />
```

Comment:
This is fine, but wouldnt it be easier to just use the params hook to get it from the url? inside the Actions component? Then you dont have to pass it each time?